test/e2e/helpers/test-utils.ts

```diff
@@ -86,15 +86,23 @@ async function chromiumAndFirefoxMousePanning({ originPoint, destinationPoint }:
   await page.mouse.up();
 }
 
-export async function chromiumZoom(xTimes: number, point: Point, deltaX: number): Promise<void> {
+export async function mouseZoom(xTimes: number, point: Point, deltaX: number): Promise<void> {
+  if (!isMouseZoomSupportedByTest) {
+    throw new Error(`Mouse zoom is not supported with ${getTestedBrowserFamily()}`);
+  }
+  await chromiumMouseZoom(xTimes, point, deltaX);
+}
+
+async function chromiumMouseZoom(xTimes: number, point: Point, deltaX: number): Promise<void> {
```

Comment:
Missing replacement in `bpmn.navigation.performance.test.ts`

Reply:
performance tests directly use the function calling the CDP API, not this one